goss-testing/scripts/check_k8s_loadbalancers.sh

```diff
@@ -105,4 +146,23 @@ else
             if [[ $ip =~ ^[0-9]+\.[0-9]+\.[0-9]+\.[0-9]+$ ]]; then echo 'PASS'; else echo 'FAIL'; fi
         fi
     done
+else
+# CHN is not configured and CAN is configured
+    if [[ ${print_results} -eq 1 ]]
+    then
+        echo 'INFO: CHN is not configured, excluding istio-ingressgateway-chn and cray-oauth2-proxies-customer-high-speed-ingress LoadBalancers'
+        echo 'INFO: CAN is configured, including CAN LoadBalancers'
+        printf '\n%-16s %-35s %-15s %-4s\n' "NAMESPACE" "LOADBALANCER" "IPADDRESS" "STATUS"
+    fi
+    kubectl get service -A | grep LoadBalancer | grep -Ev "istio-ingressgateway-chn|cray-oauth2-proxies-customer-high-speed-ingress|cmn" | awk '{print $1" "$2" "$5}' | \
```

Comment:
Also need to exclude `cray-oauth2-proxies-customer-management` when you're excluding CMN.   
And should probably exclude `uai` in the case there there is no CMN, CAN, or CHN.   But I don't know if they would even create UAIs in that case anyway.